src/classes/GS_AdminSetup.cls

```diff
@@ -36,13 +36,15 @@
 public with sharing class GS_AdminSetup {
 
     private static final String DISABLED = 'Disabled';
+    private static Set<Id> storedItemsId = null;
     /**
      * @description Returns a list of the checklist section with each subsection.
      * Each section is defined in GS_Admin_Checklist_Section__mdt and each subsection is defined in GS_Admin_Checklist_Item__mtd
```

Comment:
Typo at the end of this sentence: GS_Admin_Checklist_Item__mtd should be GS_Admin_Checklist_Item__mdt